Hello microverse branch 1

Bludidi opened the pull request:
# Set up "Hello Microverse!" project: Module 1
This project is for checking if there is:
- No linter errors. 
- Github Flow is used correctly. 
- Work is documented correctly and professionally.
- .gitignore file is added correctly and working correctly.

Bludidi committed:
Create new.html

I created a github folder in the directory

Bludidi committed:
Add files via upload

Added workflows folder

Bludidi committed:
Create test.md

Added a test.md file that is to be ignored by Git.

Bludidi committed:
Create gitignore

Bludidi committed:
Delete github directory

Bludidi committed:
Create name.md

Bludidi committed:
Delete .github directory

Bludidi committed:
Create name.md

I created a .github folder to put workflows inside it.

Bludidi committed:
Add files via upload

I have added workflows folder in this directory.

Bludidi committed:
Update styles.css

I have put a space between a selector and "{", and 2 space indentation

Bludidi committed:
Merge branch 'Hello-Microverse-branch-1' of https://github.com/Bludidi/Hello_Microverse_Projects into Hello-Microverse-branch-1

1GORDON commented on test.md:
```diff
@@ -0,0 +1 @@
+This file should be ignored by Git. 
```
Great job @Bludidi 🤝

- [x] Kindly add this file `test.md` to the `.gitignore file` so that it can be ignored by git. As of now it is appearing on git meaning it was not ignored.

Bludidi replied:
I have edited the file name I had put earlier to test.md

1GORDON commented on gitignore:
```diff
@@ -0,0 +1 @@
+Bludidi/Hello_Microverse_Projects/test.md 
```
Great job @Bludidi 🤝

- [x] You may kindly remove the rest of the text before `test.md`. Only retain `test.md` because you want git to ignore the `test.md` file.
- [x] Also kindly add the `node_modules` in this file as well. But put it on a different line.

Bludidi replied:
I have edited the file name I had put earlier to test.md  and also added node_modules in the gitignore.

1GORDON commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```
Great job @Bludidi 🤝

- [x] Great job adding the `linters.yml file`. You may also add the `.hintrc` file and `stylintrc.json` file in the root directory. The files can be found in the correct [linter files](https://github.com/microverseinc/linters-config/tree/master/html-css) provided.

Bludidi replied:
I have added .hintrc file and stylintrc.json

1GORDON reviewed (changes requested):
##
## Hi @Bludidi ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly make sure your PR description section contains the description of the tasks accomplished within this Pull request.
![description](https://user-images.githubusercontent.com/63917725/184901498-f346888e-7452-4bc7-8f6a-fa8936ed9af8.PNG)

- [ ] Kindly add a readme file to your project and customize it accordingly. You may use this [readme template](https://github.com/microverseinc/readme-template).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Bludidi committed:
Update gitignore

Edited test.md file so it can be ignored by git and added node_modules to gitignore

Bludidi committed:
Add files via upload

I have added .hintrc file and .styleintrc.json file as my code reviewer suggested.

Bludidi committed:
Update README.md

I have edited a README.md file to give a detailed insight about this code.

Bludidi committed:
Update gitignore

I have added .vscode folder in the gitignore file

ElsonOtake reviewed (changes requested):
STATUS: CHANGES REQUIRED 🟡 

Hi @Bludidi, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥲 🥲 

![woohoo-youre-almost](https://user-images.githubusercontent.com/77038610/154921145-1f7bb8f6-c843-4ba5-a534-6b840c600558.jpeg)

 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️


 ## Required Changes ♻️

- [x] As required by the previous reviewer. Kindly make sure your PR description section contains the description of the tasks accomplished within this Pull request. For more informations click [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

<img width="912" alt="Screen Shot 2022-08-17 at 9 10 44 AM" src="https://user-images.githubusercontent.com/47673311/185116382-2134722e-c91d-43ee-abf9-6c18de700b77.png">

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Bludidi committed:
Rename gitignore to .gitignore

I have edited the name of the .gitignore file from gitignore to .gitignore file.

vechicin reviewed (approved):
# Hi @Bludidi ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good job implementing the previous reviewer's requirement

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Bludidi commented:
# Branches Merged 
I have merged the Hello-Microverse-branch with the main branch, it was approved by a code reviewer.
